server/src/main/java/org/apache/druid/curator/CuratorModule.java

```diff
@@ -155,7 +155,11 @@ public EnsembleProvider makeEnsembleProvider(CuratorConfig config, ExhibitorConf
 
     RetryPolicy retryPolicy;
     if (config.getTerminateDruidProcessOnConnectFail()) {
-      // It's unknown whether or not this precaution is needed.  Tests revealed that this path was never taken.
+      /** It's unknown whether or not this precaution is needed.  Tests revealed that this path was never taken.
```

Comment:
You cannot Javadoc inner variables in a method. Please make it an ordinary comment.

Reply:
Done